include/f_core/flight/CPhaseController.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+#include <array>
+#include <cstdint>
+#include <cstdio>
+#include <zephyr/kernel.h>
+
+void flight_log_init();
+void flight_log_source_event(const char *source, const char *event);
+void flight_log_event_confirmed(const char *event, bool current_state);
```

Comment:
for logs when a Source reports an Event and the decision func returns true. Passes in whether or not the state has haappened yet so we can say so in the log